src/components/FormStart/index.js

```diff
@@ -125,15 +125,21 @@ const FormStart = ({form, onFormStart}) => {
 
         <FormStartMessage form={form} />
 
-        <LoginOptions form={form} onFormStart={onFormStart} />
+        {hasActiveSubmission ? (
+          <ExistingSubmissionOptions form={form} onFormAbort={onFormAbort} />
+        ) : (
+          <LoginOptions form={form} onFormStart={onFormStart} />
+        )}
       </Card>
     </LiteralsProvider>
   );
 };
 
 FormStart.propTypes = {
   form: Types.Form.isRequired,
+  hasActiveSubmission: PropTypes.bool,
```

Comment:
if you don't provide a default, please mark the prop as required

```suggestion
  hasActiveSubmission: PropTypes.bool.isRequired,
```